tests/wpt/metadata/fetch/nosniff/parsing-nosniff.window.js.ini

```diff
@@ -1,16 +0,0 @@
-[parsing-nosniff.window.html]
```

Comment:
Please revert these changes; the test passes intermittently and is unrelated to your changes.